sdk/lib/rtl/actctx.c

```diff
@@ -2958,8 +3017,16 @@ static NTSTATUS get_manifest_in_pe_file( struct actctx_loader* acl, struct assem
     NTSTATUS            status;
     SIZE_T              count;
     void               *base;
+    WCHAR resnameBuf[20];
+    LPCWSTR resptr = resname;
 
-    DPRINT( "looking for res %S in %S\n", resname, filename );
+    if ((!((ULONG_PTR)resname >> 16)))
```

Comment:
Too many parentheses here. :thinking: 

Also, this way to check string pointer vs. identifier looks a bit hacky.

Reply:
There is the `IS_INTRESOURCE` that does just that.

Reply:
Which is defined in winuser.h. i doubt we can use this header in ntdll

Reply:
You're correct. I was actually just pointing to binarymaster that checking string pointer vs. identifier that way is what windows does.